git-recent-branches

```diff
@@ -270,7 +275,7 @@ if __name__ == "__main__":
                         help="Show top commit on each branch. Uses git log, use git log options for formatting the output")
     parser.add_argument("--limit",
                         type=int,
-                        default=-1,
+                        default=-2,
```

Comment:
we can't call `get_configured_limit` from here?

Reply:
No can do. I need `repo` to get the config, and `repo` needs the already processed `args` thing.

Reply:
`init_repo()` just passes the args through to `find_repo()` which does... nothing with them. So we could drop that bit :)

Reply:
Ah :) Removed the `args` from `init_repo()` and `find_repo()` moved things around accordingly.